deneme3.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from py2neo import authenticate, Graph,Node

```

Comment:
Python'da main fonksiyon için genelde şu satırlar kullanılır:

def main():
    # do something

if __name__ == '__main__':
    main()
Bunun anlamını araştırabilirsin :) sonra bana da anlatırsın. (Search key: Python scope, local scope vs global scope)
Bu şekilde dosya okuma için ayrı ve neo4j için ayrı fonksiyonlar yazarak kodu daha fonksiyonel hale getirelim.

Bir de koda bol bol comment atalım. Örneğin: # Neo4j server connection, # parsing actor info from file vs..